.travis.yml

```diff
@@ -4,6 +4,8 @@ language: go
 
 go:
     - 1.7
+    - 1.8
+    - 1.9
```

Comment:
Not actually related to the change, but I noticed that we were a couple versions behind in here and added them for completeness.

Reply:
You could also add `1.x` here
https://docs.travis-ci.com/user/languages/go/#Specifying-a-Go-version-to-use

Reply:
Ah neat trick! I didn't know about it.

I think it might be slightly better to leave these other versions in the build matrix for now just so that we know for sure that we're maintaining backwards compatibility.

Reply:
Should we replace `tip` with `1.x` and leave the other explicit version numbers? IIRC, `tip` is pretty slow to run (I think it builds from source).

Reply:
Good call. Changed in 7d2dab5. Thanks guys!